servers/zms/src/main/java/com/yahoo/athenz/zms/notification/PendingGroupMembershipApprovalNotificationTask.java

```diff
@@ -88,4 +91,20 @@ public NotificationEmail getNotificationAsEmail(Notification notification) {
             return new NotificationEmail(subject, body, fullyQualifiedEmailAddresses);
         }
     }
+
+    public static class PendingGroupMembershipApprovalNotificationToMetricConverter implements NotificationToMetricConverter {
+        private final static String NOTIFICATION_TYPE = "pending_group_membership_approval";
+
+        @Override
+        public NotificationMetric getNotificationAsMetrics(Notification notification) {
+            String[] record = new String[] {
+                    METRIC_NOTIFICATION_TYPE_KEY, NOTIFICATION_TYPE
+            };
+
+            List<String[]> attributes = new ArrayList<>();
+            attributes.add(record);
+            // This notification doesn't contain any details. We should consider adding the recipients in their own tags.
```

Comment:
There are two notification tasks that don't contain details at all (PendingGroupMembershipApprovalNotificationToMetricConverter and PendingRoleMembershipApprovalNotificationToMetricConverter).
Currently, I only record that notification of that type happened. Let me know if I should record the recipients.